javascript/javascript1/week3/homework.js

```diff
@@ -0,0 +1,169 @@
+// Item array removal
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+console.log(names)
+const nameToRemove = "Ahmad";
+
+function Remove(arrayOfNames=[],name){
+   const index=arrayOfNames.indexOf(name)
+   if ([index!==-1]) {
+    arrayOfNames.splice(index, 1); 
+  }
+  
+  return(arrayOfNames)
+}
+Remove(names,`Ahmad`)
+console.log(names)
+
+
+//When will we be there??
+
+
+function travelTime(distance,speed){
+  return{
+    calculate(){
+      let time= distance/speed;
+      let hourPart=Math.floor(time);
+      let minutePart= Math.round((time-hourPart)*60);
+      return `${hourPart} hours and ${minutePart} minutes`;
+    }
+  }
+}
+const travel = travelTime(432, 50); 
+console.log(travel.calculate());
+
+
+//Series duration of my life
+
+const seriesDurations = [
+  {
+    title: "Game of thrones",
+    days: 7,
+    hours: 19,
+    minutes: 56,
+  },
+  {
+    title: "Breaking bad",
+    days: 4,
+    hours: 23,
+    minutes: 48,
+  },
+  {
+    title: "How I meet your mother",
+    days: 11,
+    hours: 0,
+    minutes: 12,
+  },
+];
+
+function logOutSeriesText()
+{
+  const lifespan= 80*365*24*60;
+ 
+  for(let i=0; i<seriesDurations.length;i++){
+   const moviTime=  ((seriesDurations[i].days)*24*60) + ((seriesDurations[i].hours)*60) +seriesDurations[i].minutes;
+    let percentOfLife = (moviTime/lifespan)*100
+    console.log(`${seriesDurations[i].title} took ${percentOfLife.toFixed(2)} of my life`)
+  }
+}
+logOutSeriesText();
+
+
+//NOnoN0nOYes (Note taking app)
+// Save a note
+
+ const notes=[];
+ function saveNote(content, id){
+  notes.push({ content, id })
+  console.log(notes)
+ }
+ saveNote(`Pick up groceries`, 1);
+saveNote(`Do laundry`, 2);
+console.log(notes)
+
+
+// get a note
+function getnote(id){
+  for(i=0; i<notes.length;i++){
+    if (notes[i].id===id)
```

Comment:
This if-statement has no braces while others do. I personally prefer sticking with always using braces as it is more consistent and you avoid gotchas. This is purely up to you though! What do you think? 😄

Reply:
yes you right , I did not use the curly braces because I had only one statement on the if but it was better to used it